ouroboros-network-framework/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 * Split `test` component into `io-tests` and `sim-tests`.
 
+* Fix Server2 sim test
```

Comment:
You should explain what is fixed / modified, e.g. this and that was modified in order to ...; or include a link to the issue.